catalog/transformer/catalog-transformer-rtf/src/main/resources/default-category-association.config

```diff
@@ -0,0 +1,6 @@
+title="Association"
```

Comment:
❓ Would it be worth it to combine the HTML and RTF service factory since they use the exact same configurations?

Reply:
Don't have a strong opinion on that save for potentially having to "untangle" them if the categories diverge in the future.